core/src/main/java/org/geysermc/geyser/translator/inventory/item/nbt/EnchantmentTranslator.java

```diff
@@ -39,6 +39,7 @@
 
 @ItemRemapper
 public class EnchantmentTranslator extends NbtItemStackTranslator {
+    private int sweepingEdge = -1;
```

Comment:
Can we find another way to handle this? This ItemTranslator will be used in multiple threads concurrently so there's a slight chance for race issues.

Reply:
not sure if the new solution is better; i'll leave this open for now